packages/replay-next/pages/variables.css

```diff
@@ -1151,3 +1151,75 @@
   --focus-mode-popout-icon-color: black;
   --focus-mode-popout-text-input-background-color: var(--theme-base-85);
 }
+
+/*
+Migration notes:
+  * removed from src/devtools/client/themes/variables.css
+  * things below this point are not organized yet
+*/
+
+/*
+ * For doorhangers elsewhere in Firefox, Mac uses fixed colors rather than
+ * system colors.
+ */
+:root[platform="mac"].theme-light {
+  --theme-popup-color: hsl(0, 0%, 10%);
+  --theme-popup-border-color: var(--grey-90-a20);
+}
+
+/* browser specific tweaks */
+@-moz-document url-prefix() {
+  :root.theme-dark {
+    --modal-bgcolor: rgba(0, 0, 0, 0.98);
+  }
+}
+
+/* Z-index scale */
+
+/* Z-index values should always be defined in #app. This allows us to at a glance determine
+relative layers of our application and prevents bugs arising from arbitrary z-index values. */
+
+/* Whenever possible, we should rely on the element stack order instead of z-index. If you're
+certain that you **have** to use a z-index (e.g. element with position: absolute/fixed), define a
+custom variable for it below so we can keep track of everything in one place. */
+
+#__next {
+  --z-index-1: 10;
+  --z-index-2: 20;
+  --z-index-3: 30;
+  --z-index-4: 40;
+  --z-index-5: 50;
+  --z-index-6: 60;
+  --z-index-7: 70;
+  --z-index-8: 80;
+  --z-index-9: 90;
+  --z-index-10: 100;
+}
+
+#__next {
+  --z-index-1--timeline-comment: var(--z-index-1);
+  --z-index-1--paused-comment: var(--z-index-1);
+  --z-index-1--paused-message: var(--z-index-1);
+  --z-index-1--comment-container: var(--z-index-1);
+  --z-index-1--icon-tooltip: var(--z-index-1);
+  --z-index-1--timeline-tooltip: var(--z-index-1);
+  --z-index-2--hovered-point-secondary: var(--z-index-2);
+  --z-index-3--hovered-point-primary: var(--z-index-3);
+  --z-index-3--hovered-comment-primary: var(--z-index-3);
+  --z-index-4--name: var(--z-index-4);
+  --z-index-5--name: var(--z-index-5);
+  --z-index-6--name: var(--z-index-6);
+  --z-index-7--name: var(--z-index-7);
+  --z-index-7--mask: var(--z-index-7);
+  --z-index-8--modal: var(--z-index-8);
+  --z-index-8--dropdown: var(--z-index-8);
+}
```

Comment:
Why are we defining these values on `#__next` rather than `:root` ? I think they should be the latter. `#__next` is too coupled to an arbitrary hosting environment detail.

Reply:
Ah, will move

Reply:
Ok, these are moved